pkg/config/service.go

```diff
@@ -33,6 +35,10 @@ type CPUCostConfig struct {
 	WebCpuCost            float64 `yaml:"web_cpu_cost"`
 }
 
+type DebugConfig struct {
+	DebugHandlerPort int `yaml:"debug_handler_port"` // Port used to launch the egress debug handler. 0 means disabled. The egress service will reserve the next 1000 port to exchange debug info with the egress handlers.
```

Comment:
could avoid extra nesting and put it with healthport/prometheus port

Reply:
Yes